src/_gettsim_tests/test_data/wohngeld/2023/skip_mischhaushalt_mit_einer_sgb_ii_bg.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 info:
-  note: https://www.bmwsb.bund.de/Webs/BMWSB/DE/themen/wohnen/wohnraumfoerderung/wohngeld/wohngeldrechner-2023-artikel.html
-  source: ''
+  note: Skipped because multiple fgs in hh.
```

Comment:
Can't we supply the `bg_id` or would that run counter the purpose of the test in the first place? Same for the next test.